source/ContextualPopupDecorator.js

```diff
@@ -126,6 +127,15 @@
 		*/
 		requestHidePopup: function () {
 			this.waterfallDown('onRequestHidePopup');
+		},
+
+		/**
+		* When back key is pressed, return spotlight focus to popup button
+		*
+		* @private
+		*/
+		requestSpot: function (inSender, inEvent) {
+			enyo.Spotlight.spot(this);
```

Comment:
Yes, you are right. I missed it!
I updated it and testified its functionality on PC.
Thanks @clinuz